addons/bus/static/src/js/services/async_service.js

```diff
@@ -0,0 +1,41 @@
+odoo.define('bus.asyncService', function (require) {
+"use strict";
+
+const AbstractService = require('web.AbstractService');
+const { serviceRegistry, bus, _lt, _t } = require('web.core');
+
+
+const notifHead = _lt("Background task created");
+const notifCreatedBody = _lt("The task %s has been scheduled for processing and will start shortly.");
+
+const asyncService = AbstractService.extend({
+    dependencies: ['bus_service', 'ajax', 'crash_manager'],
+    start() {
+        this._super(...arguments);
+        this.call('bus_service', 'onNotification', this, this._onNotification);
+        bus.on('async_job_created', this, (jobId, jobName) => {
+            this.do_notify(notifHead, _.str.sprintf(""+notifCreatedBody, _t(jobName)));
+        });
```

Comment:
- same comments as before (explicit use of `toString()`, ignore `_t(<variable>)`)
- leave listening on bus to `AsyncJob(Systray)Menu`, prefer `do_notify()` right when handling notification in this service.